src/amp-story-player/amp-story-player-impl.js

```diff
@@ -98,6 +98,10 @@ const STORY_MESSAGE_STATE_TYPE = {
   MUTED_STATE: 'MUTED_STATE',
 };
 
+/** @typedef {{ state:string, value:boolean }} */
+// eslint-disable-next-line no-unused-vars
```

Comment:
I think the convention is to end the variable name with `Def` and then the lint check will automatically be skipped

Reply:
Thank you! This worked.